sql/000006.sql

```diff
@@ -0,0 +1,6 @@
+create index jobs_priority_run_at_id_locked_at_without_failures_idx
+  on :GRAPHILE_WORKER_SCHEMA.jobs (priority, run_at, id, locked_at)
+  where attempts < max_attempts;
```

Comment:
Given `locked_at` is after `id` in the index (which is unique), I feel like it's unlikely to give much value adding it to the index, but it will increase both churn on the index and the size of the index. How about we just do the partial index thing; does that still improve performance in your use case? It'll actually reduce index size which is a great addition; this could really make a difference in the case of lots of perma-fails.

```suggestion
create index jobs_priority_run_at_id_alive_idx
  on :GRAPHILE_WORKER_SCHEMA.jobs (priority, run_at, id)
  where attempts < max_attempts;
```